### PR TITLE
testing again container instance

### PR DESCRIPTION
**Pull request by zikalino**

### CODE REVIEW

File: tests/integration/targets/azure_rm_containerinstance/tasks/main.yml [hunk 1]
```diff
@@ -13,25 +13,21 @@
...
-        memory: 1.5
   register: output
 
 - name: Assert the container instance is well created
   assert:
     that:
-      - output.changed
-      - output.provisioning_state == 'Creating'
+      #- output.changed
+      - output.provisioning_state == 'Succeeded'
```
Comment (yungezz) [inline tests/integration/targets/azure_rm_containerinstance/tasks/main.yml]: is it able to assert multiple expression with 'or', to loose test pass criteria

Comment (zikalino) [reply tests/integration/targets/azure_rm_containerinstance/tasks/main.yml]: ooops.... i think i made mistake here

Comment (zikalino) [reply tests/integration/targets/azure_rm_containerinstance/tasks/main.yml]: # should be removed

**Comment by yungezz** [comment]
will check whether this pr is still valid  after sync upstream to role

**Pull request closed without merging**